modules/installing-rhv-requirements.adoc

```diff
@@ -30,6 +30,9 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 *** 16 GiB or more for each of the three control plane machines which provide the control plane.
 *** 16 GiB or more for each of the three compute machines, which run the application workloads.
 * The {rh-virtualization} storage domain must meet link:https://access.redhat.com/solutions/4770281[these etcd backend performance requirements].
+* For affinity group support:
+** Production setup: For hard anti-affinity, you need a minimum of three physical machines. For more than three worker nodes, one physical machine per worker or control plane. Workers and control planes can be on the same physical machine. For example, if you have three workers and three control planes, you need three physical machines. If you have four workers and three control planes, you need four physical machines.
```

Comment:
I think this section should be changed.
On "Installing a cluster quickly on RHV" you will always get 2 affinity groups one for the worker and one for the control plane, so the requirement on the  "Installing a cluster quickly on RHV" should be simple and clear that we require 3 at least hosts to exist in the RHV cluster because of affinity groups, and a note saying to disable affinity groups refer to the customization page.

On "Installing a cluster on RHV with customizations" things are more complex:

- if the user chooses to use the default affinity groups setup then we should state that we require a minimum of 3 hosts in the RHV cluster and another host per worker (i.e if the user set up a cluster with 4 workers then he needs 4 hosts in the cluster)
- if the user chooses to disable affinity groups then there is no special requirement but we should advise against that for production setup (not in the requirements section)
- If the user chooses to configure custom affinity groups then he needs to see that the affinity group rules that he defined make sense for the RHV resources.

Other than the part of the requirements needs to be different per page, I think that separating Production vs Non-production setup on affinity groups is confusing especially when it is on the requirement section... maybe we should have an affinity group recommendations or just an explanation on the docs :/

Reply:
@janoszen what do you think?

Reply:
Also I noticed that because this was changed in a requirements section that is shared between all pages then it appears on the UPI section and restricted network... that is not the case since there are no affinity groups in UPI

Reply:
I think that anyone who is worth their salt as a sysadmin will figure out how to do stuff. This is just for the people who don't really have a clue and it's a safe default. LGTM

Reply:
>     * If the user chooses to configure custom affinity groups then he needs to see that the affinity group rules that he defined make sense for the RHV resources.
> maybe we should have an affinity group recommendations *or just an explanation on the docs :/*
I just scanned the RHV Admin Guide (nothing about affinity) the VMM guide (this is where affinity groups are documented) and the Planning and Prereqs Guide (nothing about affinity) I see nothing in any of these places about technical requirements for using affinity groups.

Reply:
@Gal-Zaidman Take a look at my edits here. I believe I responded to all your concerns.

Reply:
Yep looks good to me now